source/renderer/app/components/widgets/Slider.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import BigNumber from 'bignumber.js';
+import RcSlider from 'rc-slider';
+import { TooltipSkin } from 'react-polymorph/lib/skins/simple/TooltipSkin';
+import { Tooltip } from 'react-polymorph/lib/components/Tooltip';
+import { shortNumber } from '../../utils/formatters';
+import styles from './Slider.scss';
+
+type Props = {
+  className?: string,
+  min: number,
+  max: number,
+  marks?: any,
+  step?: number,
+  vertical?: boolean,
+  handle?: Function,
+  included?: boolean,
+  reverse?: boolean,
+  disabled?: boolean,
+  dots?: boolean,
+  onBeforeChange?: Function,
+  onChange: Function,
+  onAfterChange?: Function,
+  minimumTrackStyle?: any,
+  maximumTrackStyle?: any,
+  handleStyle?: any,
+  trackStyle?: any,
+  railStyle?: any,
+  dotStyle?: any,
+  activeDotStyle?: any,
+  defaultValue?: number,
+  value: number,
+  showTooltip?: boolean,
+  minTooltip?: string,
+  maxTooltip?: string,
+  startPoint?: number,
+  tabIndex?: number,
+  ariaLabelForHandle?: string,
+  ariaLabelledByForHandle?: string,
+  ariaValueTextFormatterForHandle?: Function,
+};
+
+@observer
+export default class Slider extends Component<Props> {
+  static defaultProps = {
+    min: 0,
+    max: 100,
+    value: 0,
+  };
+
+  render() {
+    const { showTooltip, minTooltip, maxTooltip, ...rest } = this.props;
+    const { min, max, value } = rest;
+
+    const valueMarkLeftPosition =
+      max === min ? `0` : `${((value - min) / (max - min)) * 100}%`;
+    const valueMarkStyle = { left: valueMarkLeftPosition };
+    const formattedValue = new BigNumber(value).toFormat(0);
+
+    return (
+      <div className={styles.component}>
+        <div className={styles.upperMarks}>
+          <div className={styles.minMark}>
+            {showTooltip ? (
+              <Tooltip skin={TooltipSkin} tip={minTooltip}>
+                {shortNumber(min)}
+              </Tooltip>
+            ) : (
+              <>{shortNumber(min)}</>
```

Comment:
@yakovkaravelov Do we really need React Fragment here and in line `80`?

Reply:
@tomothespian React Fragment doesn't have actual dom elements so above code doesn't have any negative effect.
I put it to make it consistent look with tooltip case. Removing it is really not needed actually.
But, I will delete it for you.

Reply:
@yakovkaravelov ok, if the fragment is not necessary then is better to remove it. Thanks!